proto/cosmos/gov/v1/genesis.proto

```diff
@@ -17,10 +17,6 @@ message GenesisState {
   repeated Vote votes = 3;
   // proposals defines all the proposals present at genesis.
   repeated Proposal proposals = 4;
-  // params defines all the paramaters of related to deposit.
-  DepositParams deposit_params = 5;
-  // params defines all the paramaters of related to voting.
-  VotingParams voting_params = 6;
-  // params defines all the paramaters of related to tally.
-  TallyParams tally_params = 7;
+  // params defines all the paramaters of x/gov module.
+  Params params = 5;
```

Comment:
because we are not bumping the proto file versions this would need to be number 8 and the other numbers would be reserved.